index.js

```diff
@@ -147,6 +147,13 @@ var scssContextParser = (function () {
           end : lineNumberFor(endIndex) + 1
         };
       }
+    } else {
```

Comment:
This is assuming every line that contains `{` and that is not matched by the SCSS types selector above is a block of CSS rules.

I'm not sure this is really bulletproof, and would like to double check this. What about `@keyframes` rules for example? Or `@media` and nested rules?

That was probably one of the reasons we didn't want to allow to comment CSS classes. :smile: